packages/common/components/blocks/leadership-company-videos.marko

```diff
@@ -1,12 +1,18 @@
 import { get, getAsArray } from "@parameter1/base-cms-object-path";
 import queryFragment from "../../graphql/fragments/leadership-company-videos";
```

Comment:
To remove any confusion about this, rename these fragments to `../../graphql/fragment-factories/<filename>`. Any file requiring the fragment will need to be updated, and this will be a quick indicator to future us as well.

Also rename your imported variables (like `fragmentFactory`, `fragmentBuilder`, etc) to indicate this:
```js
import fragmentFn from "../../graphql/fragment-factories/leadership-company-videos";

const { site } = out.global;
const queryFragment = fragmentFn(site.get("leaders.alias"));
// ...
```

This will affect a large number of files, but I'd rather be explicit with the changes we're making than leave it ambiguous.